pkg/tfgen/generate_nodejs.go

```diff
@@ -1078,6 +1090,14 @@ func (g *nodeJSGenerator) gatherCustomImports(mod *module, info *tfbridge.Schema
 	return nil
 }
 
+func (g *nodeJSGenerator) getDeprecationDetails(message string) string {
+	if message != "" {
+		return fmt.Sprintf("/** @deprecated %s%s*/", message, " ")
```

Comment:
Why the second `%s` for `" "`?

Reply:
Any reason not to just inline this?  We're effectively just doing the `if` twice in all the locations above - right?

Reply:
done!